pages/web_cloud/email_and_collaborative_solutions/mx_plan/how_to_configure_android/guide.en-gb.md

```diff
@@ -1,97 +1,176 @@
 
-If you need to enter any settings manually in your account preferences, the technical settings you will need to use with our MX Plan solution are listed below:
+When you configure your mail client, you must enter the information from the **incoming server** to receive the emails, and the **outgoing server** to send the emails. To send emails, there is no choice, the **SMTP** (**S**imple **M**ail **T**ransfer **P**rotocol) is used. For the reception, you will have the choice between **POP** or **IMAP**.
 
-- **for IMAP configuration**
+![mxplan](images/mxplan-android-popimap-01.png){.thumbnail}
 
-|Server type|Server name|Security type|Port|
-|---|---|---|---|
-|Incoming|ssl0.ovh.net|SSL/TLS|993|
-|Outgoing|ssl0.ovh.net|SSL/TLS|465|
+To understand the difference between POP and IMAP, we will break down the elements that make up the processing of your incoming emails:
 
-- **for POP configuration**
+1. **Your device**: a computer, smartphone or tablet. This is your consultation support.
 
-|Server type|Server name|Security type|Port|
-|---|---|---|---|
-|Incoming|ssl0.ovh.net|SSL/TLS|995|
-|Outgoing|ssl0.ovh.net|SSL/TLS|465|
+2. **Your email client**: software or application dedicated to managing your emails. Your choice will determine the level of usability and functionality you will need to view your emails.
 
-### Step 2: Use the email address
+3. **The receiving protocol**: a choice that determines how you check emails on your device. This affects other devices that access the same email account.
+    - **IMAP**: Your email client queries the email server and downloads the emails to your device. When you view an unread email, the server marks it as “read” by default. Other devices configured in IMAP will be able to check this status and view this email until it has been deleted on one of the devices.
+    - **POP**: Your email client requests the email server and downloads the emails to your device. By default, once the email is downloaded to your device, the message is deleted from the server. As a result, other devices connected to this email address will not be able to view this email.
 
-Once you have configured your email address, you can start using it! You can now send and receive emails.
+![mxplan](images/mxplan-android-popimap-02.png){.thumbnail}
 
-OVHcloud also has a web application you can use to access your email account from your browser, via the following address: <https://www.ovh.co.uk/mail/>. You can log in using your email credentials.
+> [!primary]
+>
+> This description is a summary, it represents the standard operation of these two protocols. You can set the POP so that emails are not deleted when you check your emails. The goal here is to describe how these two protocols work natively and avoid any additional steps to match your needs.
```

Comment:
```suggestion
> This description is a summary, it represents the standard operation of these two protocols. You can configure the POP so that emails are not deleted when you check your emails. The goal here is to describe how these two protocols work natively and avoid any additional steps to match your needs.
```